RoadMap_SwiftUI/RoadMap_SwiftUI/ViewModel/ClimateViewModel.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ClimateViewModel.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 31.01.2023.
+//
+
+import SwiftUI
+
+/// Вью модель экрана климат контроля
+final class ClimateViewModel: ObservableObject {
+    @Published var isAcIsOn = false
```

Comment:
```suggestion
// MARK: - Public Property
    @Published var isAcIsOn = false
```
```suggestion
    @Published var isAcOn = false
```
Два is кажется не нужно